src/test/java/net/sf/jabref/logic/importer/BibDatabaseTestsWithFiles.java

```diff
@@ -0,0 +1,46 @@
+package net.sf.jabref.logic.importer;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.importer.fileformat.BibtexParser;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.preferences.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BibDatabaseTestsWithFiles {
+
+    private ImportFormatPreferences importFormatPreferences;
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance(); // set preferences for this test
+
+        importFormatPreferences = ImportFormatPreferences.fromPreferences(Globals.prefs);
+    }
+
+    @Test
+    public void resolveStrings() throws IOException {
+        try (FileInputStream stream = new FileInputStream("src/test/resources/net/sf/jabref/util/twente.bib");
```

Comment:
I would prefer if you just create the database with the strings by hand (and split the test into 4)